### ISSUE TITLE
Resolve existing FIXME & TODO items in NeXus file docs

### ISSUE THREAD

**Issue by prjemian** [prjemian/spec2nexus#265]
Existing FIXME & TODO items now:

```bash
(bluesky_2022_1) prjemian@zap:~/.../prjemian/spec2nexus$ grep FIXME docs/source/nexus_layout/*.rst
docs/source/nexus_layout/general_tree.rst:FIXME: links have changed direction, new contents added
(bluesky_2022_1) prjemian@zap:~/.../prjemian/spec2nexus$ grep TODO docs/source/nexus_layout/*.rst
docs/source/nexus_layout/mca.rst:TODO: write
docs/source/nexus_layout/mca.rst:TODO: show with one MCA
docs/source/nexus_layout/mca.rst:TODO: show with CALIB
docs/source/nexus_layout/metadata.rst:TODO:  #U and other (#H/#V, #UXML, ...)
docs/source/nexus_layout/sample.rst:TODO
docs/source/nexus_layout/temperature.rst:TODO: ``#X`` (example calib.dat, scan 11) - Temperature Set Point
```

_Originally posted by @prjemian in https://github.com/prjemian/spec2nexus/issues/192#issuecomment-1068486353_

**Comment by prjemian** [comment]
The list is larger than reported above:

```bash
(bluesky_2022_2) prjemian@zap:~/.../prjemian/spec2nexus$ git grep FIXME:
docs/source/nexus_layout/general_tree.rst:.. FIXME: links have changed direction, new contents added
src/spec2nexus/tests/test_diffractometers.py:        # [TEST_DATA_DIR, 'test_3_error.spec', -1, 'spec'],                  # FIXME: #UXML, plugin has error
(bluesky_2022_2) prjemian@zap:~/.../prjemian/spec2nexus$ git grep TODO:
.flake8:# TODO: re-enable E501 error
conda-recipe/run_test.py:# TODO: test something
demo/usaxs_uascan.py:                # TODO: remove any data where Y <= 0 (can't plot on log scale)
docs/source/nexus_layout/mca.rst:TODO: write
docs/source/nexus_layout/mca.rst:TODO: show with one MCA
docs/source/nexus_layout/mca.rst:TODO: show with CALIB
docs/source/nexus_layout/metadata.rst:TODO:  #U and other (#H/#V, #UXML, ...)
docs/source/nexus_layout/temperature.rst:TODO: ``#X`` (example calib.dat, scan 11) - Temperature Set Point
docs/source/specplot_custom_scan_macro.rst:.. TODO: describe the parts of the custom code.
src/spec2nexus/diffractometers.py:            for key in "alpha beta azimuth omega".split():  # TODO: generalize
src/spec2nexus/plugins/spec_common.py:            # TODO: what if header date is greater than this scan's date?
src/spec2nexus/plugins/spec_common.py:            raw = ""  # TODO: what default content to use?
src/spec2nexus/plugins/spec_common.py:            # TODO: replace beginning() with new algorithm
src/spec2nexus/plugins/spec_common.py:    # TODO: Needs an example data file to test (issue #8)
src/spec2nexus/plugins/spec_common.py:            sdf_object.O = []  # TODO: What if motor names are different?
src/spec2nexus/plugins/spec_common.py:    # TODO: need more examples of MCA spectra in SPEC files to improve this
src/spec2nexus/plugins/uim.py:# TODO: for each ControlLineBase, describe where data goes, both internally and in HDF5 file
src/spec2nexus/specplot.py:    # TODO: apply property and setter decorations
src/spec2nexus/specplot.py:# class NeXusPlotter(ImageMaker):    # TODO: issue #92
src/spec2nexus/specplot_gallery.py:                for k in plot_list:  # TODO: needs unit test
src/spec2nexus/specplot_gallery.py:    # TODO: do not start this process if it is running from previous call
src/spec2nexus/tests/test_spec.py:    # TODO: apply file-specific tests (see README.txt)
src/spec2nexus/tests/test_spec.py:    # TODO: apply file-specific tests (see README.txt)
src/spec2nexus/tests/test_spec.py:    # TODO: apply file-specific tests (see README.txt)
src/spec2nexus/tests/test_spec.py:    assert len(sfile.headers) == 2  # TODO: test more here!
src/spec2nexus/tests/test_spec.py:    # TODO: apply file-specific tests (see README.txt)
src/spec2nexus/tests/test_spec.py:    # TODO: apply file-specific tests (see README.txt)
src/spec2nexus/tests/test_specplot_gallery.py:    # TODO: test contents of mtime_cache.txt?
src/spec2nexus/tests/test_specplot_gallery.py:    # TODO: #69: look for handling of scan 15
src/spec2nexus/tests/test_specplot_gallery.py:    # TODO: #69: look for handling of scan 1
src/spec2nexus/tests/test_specplot_gallery.py:    # TODO: #69: look for handling of scan 1
src/spec2nexus/tests/test_specplot_gallery.py:    # TODO: look for that scan in index.html?
src/spec2nexus/tests/test_specplot_gallery.py:    # TODO: #69: look for handling of scan 5
src/spec2nexus/tests/test_writer.py:    # TODO: test writer's various functions and methods
src/spec2nexus/writer.py:                #     pass    # TODO: and do what?
src/spec2nexus/writer.py:        # TODO: refactor to use NeXus data model: signal, axes, data
```

**Comment by prjemian** [comment]
MCA documentation will be addressed in #73.
